mmpose/models/heads/regression_heads/dekr_head.py

```diff
@@ -0,0 +1,605 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Not sure that the DEKR head belongs to regression heads because it predict roots and offsets in a heatmap-based manner. Maybe it's more like a heatmap head or should be put into a new category like 'single-stage head' or 'hybrid head'?

Reply:
The localization of instance roots is based on heatmaps but the keypoint coordinates are calculated via regressed offset. In my perspective, the dense **offset** map is dissimilar from heatmap whose **activation** plays a key role. 

Maybe 'hybrid head' is a better choice than 'single-stage head' because some single-stage methods such as CID use only heatmap. It might be hard to categorize them if the latter is adopted.